src/libi_sharing/models.py

```diff
@@ -1,3 +1,59 @@
 from django.db import models
 
-# Create your models here.
+from libi_common.models import BaseModel
+
+
+class Sharing(BaseModel):
+    title = models.CharField(blank=False, max_length=200)
+    description = models.TextField(blank=True)
+    section = models.ForeignKey('Section', null=True, on_delete=models.SET_NULL)
+    category = models.ForeignKey('Category', null=True, on_delete=models.SET_NULL)
+    created_account = models.ForeignKey('libi_account.Account', null=True, on_delete=models.SET_NULL)
+    goal_price = models.IntegerField()
+    sharing_type = models.IntegerField(null=False)
```

Comment:
```suggestion
    sharing_type = models.IntegerField()
```
Django Field Class Signature를 보시면 `null=False` 는 기본값입니다. 생략해주세요.

Reply:
수정됐습니다~